snap7/client.py

```diff
@@ -643,6 +643,60 @@ def wait_as_completion(self, timeout: int) -> int:
         check_error(result, context="client")
         return result
 
+    def as_read_area(self, area, dbnumber, start, size):
+        """This is the main function to read data from a PLC.
+        With it you can read DB, Inputs, Outputs, Merkers, Timers and Counters.
+
+        :param area: chosen memory_area
+        :param dbnumber: The DB number, only used when area= S7AreaDB
+        :param start: offset to start writing
+        :param size: number of units to read
+        """
+
+        wordlen, data = self._as_read_area_prepare(area, size)
+        logger.debug(f"reading area: {area} dbnumber: {dbnumber} start: {start}: amount {size}: wordlen: {wordlen}")
+        result = self._library.Cli_AsReadArea(self._pointer, area, dbnumber, start, size, wordlen, byref(data))
+        check_error(result, context="client")
+        return data
+
+    @staticmethod
```

Comment:
this class is already quite huge, maybe just move this out of the class or into utils

Reply:
I guess utils will be a good place. Will be done.

Reply:
btw, if this is the only place where it is used then why not just put it in the function, no reason to split things up too much.